ansible/roles/sonic-common/tasks/platform.yml

```diff
@@ -9,3 +9,10 @@
   become: true
   include: platform-mlnx.yml
   when: sonic_hwsku == "ACS-MSN2700"
+
+# Install Platform Modules for Cavium
+- name: Confirm Platform Modules for Cavium are installed
+  become: true
+  include: platform-cavm.yml
+  when: sonic_hwsku == "AS7512"
```

Comment:
If you're going to add more platforms, you should consider changing condition to `sonic_hwsku in cavium_hwskus`